src/phes_base.cpp

```diff
@@ -145,14 +145,51 @@ BigModel BigModel_init(GridSquare sc){
 	return big_model;
 }
 
+double calculate_power_house_cost(double power, double head){
+	return powerhouse_coeff*pow(MIN(power,800),(power_exp))/pow(head,head_exp);
+}
+
+double calculate_tunnel_cost(double power, double head, double seperation){
+	return ((power_slope_factor*MIN(power,800)+slope_int)*pow(head,head_coeff)*seperation*1000)+(power_offset*MIN(power,800)+tunnel_fixed);
+}
+
 void set_FOM(Pair* pair){
+	bool double_half = false;
 	double seperation = pair->distance;
 	double head = (double)pair->head;
 	double power = 1000*pair->energy_capacity/pair->storage_time;
-	double power_house_cost = powerhouse_coeff*pow(MIN(power,800),(power_exp))/pow(head,head_exp);
-	double tunnel_cost = ((power_slope_factor*MIN(power,800)+slope_int)*pow(head,head_coeff)*seperation*1000)+(power_offset*MIN(power,800)+tunnel_fixed);
-	double power_cost = 0.001*(power_house_cost+tunnel_cost)/MIN(power, 800);
-	double energy_cost = dam_cost*1/(pair->water_rock*generation_efficiency * usable_volume*water_density*gravity*pair->head)*J_GWh_conversion/cubic_metres_GL_conversion;
+	double energy_cost = dam_cost*1/(pair->water_rock*generation_efficiency * usable_volume*water_density*gravity*head)*J_GWh_conversion/cubic_metres_GL_conversion;
+	double power_cost;
+	double tunnel_cost;
+	double power_house_cost;
+	if (head > 800) {
+		if (double_half) {
+			power_house_cost = 2*calculate_power_house_cost(power/2, head/2);
+			tunnel_cost = 2*calculate_tunnel_cost(power/2, head/2, seperation);
+			power_cost = 0.001*(power_house_cost+tunnel_cost)/MIN(power, 800);
+		}
```

Comment:
Will double_half ever be true? Also, do we want to allow higher than 1600? e.g. an 1800m pair is modelled as 3 600m pairs?

Reply:
Ohhh, I just saw the comment in the PR, there should definitely be a comment in the code (and should probably be something more like `#DEFINE LINEAR_EXTRAPOLATE false` to make it clearer

Reply:
Only if you change it in the file, should I make it a variable instead? I thought we should chat to Andrew before I went to that effort. Very happy to do up to 2400m - ask Andrew about that also?

Reply:
Oops I missed your second comment, will do